packages/auto-approve/src/auto-approve.ts

```diff
@@ -86,12 +88,13 @@ async function evaluateAndSubmitCheckForConfig(
     codeOwnersFile
   );
 
+  // In the case that auto-approve doesn't exist, we don't want to submit a check at all
+  // We also want to return false so that no further checks are made
+  if (isAutoApproveCorrect === 'Skip') {
```

Comment:
Thanks for the suggestion! I ended up throwing an error in the 'skip' situation, since I think that's a bit closer to the functionality we actually want.